spec/system/index_spec.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe "Index page", type: :system do
```

Comment:
do a `bundle exec rspec` you have requests specs failing because its pointing to wrong URL, since you removed it.

Reply:
Thank you! I didn't notice that failing spec.